topology/src/main/resources/proto/topology.proto

```diff
@@ -68,4 +91,11 @@ enum State {
   LEFT = 4;
 }
 
+enum ChangeStatus {
+  CHANGE_STATUS_UNKNOWN = 0;
```

Comment:
:upside_down_face: Just `UNKOWN` would be cleaner, same as for `State` above.

Reply:
If I remember correctly, protobuf was complaining when there two different enums in the same file with same name. Let me check again.

Reply:
Oh okay, how annoying :(

Reply:
`Note that enum values use C++ scoping rules, meaning that enum values are siblings of their type, not children of it.  Therefore, "UNKNOWN" must be unique within "topology_protocol", not just within "ChangeStatus".`